### ISSUE TITLE
Shopping cart widget from header causing double scrollbar + extra white space after footer when too many items in cart

### ISSUE THREAD

**Issue by zoliszabo** [woocommerce/storefront#559]
If the height of the cart widget is larger than the height of the page (because of too many products in the cart), a second scrollbar appears on the `<div id="page">` element. This happens only when the product list is hidden. See attached screen recording.
![storefront_cart_widget_double_scrollbar_bug](https://cloud.githubusercontent.com/assets/9949042/23999489/418484ac-0a61-11e7-94dc-96a3a317a2dd.gif)

**Comment by jameskoster** [comment]
Hmm that's strange, the `overflow` css property should take care of this. I just tested locally with 30 items in my cart and couldn't replicate the issue. Which browser are you using?

If you could share a link to the site for me to inspect that would be really helpful as well.

Cheers

**Comment by zoliszabo** [comment]
I used the demo store: https://demo.woothemes.com/storefront/, just deleted the sidebar widgets (because the cart widget from the sidebar also expands further with each product added to the cart and prevents the page being shorter than the cart widget from the header in its hidden state).

**Comment by jameskoster** [comment]
I see.

I followed your steps to reproduce and still wasn't able to. Please confirm which browser you're using.

**Comment by zoliszabo** [comment]
Sorry for not mentioning the browser in my previous post - I tried in Mozilla Firefox and Google Chrome (both up-to-date) on Windows 10. I'm thinking it's the same in other browsers too, I was just lazy to put ~30 products in the cart.

Here's how to reproduce it:
1. go to https://demo.woothemes.com/storefront/;
2. put about distinct 30 products in the cart;
3. go to the "Pricing Tables" page (https://demo.woothemes.com/storefront/pricing-tables/) - this is the shortest page I've found on the demo store;
4. Observe the double scrollbar.

As some custom installations of the Storefront theme may have short pages, this is an issue imho. I will also submit a possible patch in 1-2 days.
